tests/python/unittest/test_roundtrip_runtime_module.py

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" Test roundtrip of runtime modules """
+# pylint: disable=missing-docstring
+
+import pytest
+import tvm
+import tvm.testing
+from tvm import relay
+from tvm.meta_schedule.testing.custom_builder_runner import (
+    build_relay_with_tensorrt,
+)
```

Comment:
given this depends on relay, let us move to test/relay

Reply:
actually, on testcase, instead of attaching a byoc, let us just compile a cuda module and attach as extra_mods, this would also cover recursive import(as cuda contains host and device modules)

Reply:
Thanks for the suggestion, the testcase is added.